src/progressBar.js

```diff
@@ -18,7 +18,7 @@ class ProgressBarController {
     }
 
     // Don't do any of this while testing
-    if (process.env.NODE_ENV === "test") {
+    if (process.env.NODE_ENV === "lerna-test") {
```

Comment:
it's irrelevant to this PR anyway, just a quirk i noticed after i unset `NODE_ENV` on circle. probably best to add a `FIXME` and sort this out later, since it's not that important